### PR TITLE
Add Jinja + Python support.

### PR DESCRIPTION
**Pull request by lukesneeringer**
This commit adds a Jinja / Python permutation, triggered by `*.py.j2` or `*.py.jinja2`, to match the other permutations.

### CODE REVIEW

File: syntaxes/jinja-python.tmLanguage.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "jinja-python",
+    "scopeName": "text.python.jinja",
```
Comment (samuelcolvin) [inline syntaxes/jinja-python.tmLanguage.json]: This should be `source.python.jinja`

Comment (samuelcolvin) [reply syntaxes/jinja-python.tmLanguage.json]: Hi @lukesneeringer, would be great if you could fix this.

Comment (lukesneeringer) [reply syntaxes/jinja-python.tmLanguage.json]: Whups, sorry, I lost track of this PR. Fixing now.

**Comment by samuelcolvin** [comment]
Thanks, merged and released.

Review [here](https://marketplace.visualstudio.com/items?itemName=samuelcolvin.jinjahtml#review-details) would be great btw.

**Pull request merged**